contracts/acl/ACL.sol

```diff
@@ -1,3 +1,6 @@
+/* solium-disable function-order */
```

Comment:
Unsuccessfully tried to make this rule only apply to the `initialize(...)` function :( 

We can place this comment right before `contract ACL ...` instead of the beginning of the file, which looks weirder. Ideally we could create a custom solium rule that considers the initialize function as a constructor.

Reply:
Yes... we could write our own mini solium plugin for this, but ideally if you apply `solium-disable-next-line function-order`, it'd take the next line's function out of consideration for that rule.

Reply:
We'll see what happens: https://github.com/duaraghav8/Solium/issues/235